weblogic/tests/README.md

```diff
@@ -0,0 +1,16 @@
+# WebLogic Dev Readme
+
+## DDev E2E
+If you are spinning up a E2E for WebLogic for the first time, you will need to go to the Oracle Container Registry to accept the Oracle Standard Terms and Restrictions before you can successfully pull the image from the registry. 
+
+1. Login to your account on https://container-registry.oracle.com
```

Comment:
Is this a datadog account?

Reply:
It can be your personal Oracle account or a Datadog Oracle account (not Datadog SSO). I'll add a note about creating an Oracle account if that's desired.